test/util/bitcoin-util-test.py

```diff
@@ -20,40 +20,44 @@
 import subprocess
 import sys
 
+
 def main():
     config = configparser.ConfigParser()
-    config.read_file(open(os.path.dirname(__file__) + "/../config.ini"))
+    config.read_file(open(os.path.join(os.path.dirname(__file__), "../config.ini")))
 
     parser = argparse.ArgumentParser(description=__doc__)
-    parser.add_argument('-v', '--verbose', action='store_true')
+    parser.add_argument("-v", "--verbose", action="store_true")
     args = parser.parse_args()
     verbose = args.verbose
 
     if verbose:
         level = logging.DEBUG
     else:
         level = logging.ERROR
-    formatter = '%(asctime)s - %(levelname)s - %(message)s'
+    formatter = "%(asctime)s - %(levelname)s - %(message)s"
     # Add the format/level to the logger
     logging.basicConfig(format=formatter, level=level)
 
-    bctester(config["environment"]["SRCDIR"] + "/test/util/data", "bitcoin-util-test.json", config["environment"])
+    bctester(os.path.join(config["environment"]["SRCDIR"], "test/util/data"), "bitcoin-util-test.json",
+             config["environment"])
```

Comment:
Changing this line solely to split it into two causes difficulties grepping in the future.

Reply:
... and most of our python files don't adhere to the 79 char max line length (which I think is a good thing!)